app/src/ui/lib/popover.tsx

```diff
@@ -225,22 +225,32 @@ export class Popover extends React.Component<IPopoverProps, IPopoverState> {
    * The correct semantics are that a dialog element (which this is) should have
    * an aria-labelledby for it's title.
    *
-   * However, macOs Ventura introduced a regression in that the aria-labelledby
-   * is not announced and if provided prevents the aria-describedby from being
-   * announced. Thus, this method will use aria-describedby instead of the
-   * aria-labelledby for macOs Ventura. This is not semantically correct tho,
-   * hopefully, macOs will be fixed in a future release. The issue is known for
-   * macOS versions 13.0 to the current version of 13.5 as of 2023-07-31.
+   * However, macOs VoiceOver is not reliable so we have some workarounds...
    */
   private getAriaAttributes() {
-    if (!isMacOSVentura()) {
+    if (isMacOSVentura()) {
```

Comment:
Decided to invert the checks since now we have multiple. Basically the bottom return statement is the "expected" one if we do not have any `if` that redirect it.